configs/config.ini

```diff
@@ -75,14 +75,6 @@ keep_volume_storage = False
 # b/kb/mb/gb - a multiple of the unit byte for glance image info transfer
```

Comment:
or no, we need it, it's a template with some variables: `<src_cinder_db_host>` and etc...